addons/mail/static/src/core/common/message.js

```diff
@@ -366,6 +372,16 @@ export class Message extends Component {
         return true;
     }
 
+    async onClickCopy() {
+        const { model, id: threadId } = this.message.thread || {};
+        const { id: messageId } = this.message;
+        const pathname = `/mail/${model}/${threadId}/message/redirect/${messageId}`;
+        await browser.navigator.clipboard.writeText(`${getOrigin()}${pathname}`);
+        this.notification.add(_t("The message link was copied to your clipboard!"), {
```

Comment:
I don't like UI being more verbose than necessary, it gives me less incentive to read it and it gives fatigue. I'm pretty sure many users feel the same.

I would just word it `Message Link Copied!`.